content/en/community/contributing/contribution.md

```diff
@@ -0,0 +1,30 @@
+--- 
+title: "Contributing to the CHT" 
+linkTitle: "Contributions"
+weight: 3 
+description: > 
+
+  What is a contribution?
+  
+---
+
+A contribution is any meaningful action: technical or non-technical—that advances the CHT’s mission of improving community health. All contributions must add measurable value, adhere to our [Code of Conduct](https://docs.communityhealthtoolkit.org/community/contributing/code-of-conduct/), and align with the CHT’s core goals.
+
+**Ways to Contribute**
+
+There are many paths to contribute to our community. Here are some of the key ways you can get involved:
+- **Technical contributions** include:  
+  - **Code:** Submit bug fixes, develop new features, or build example apps.  
+  - **Design:** Create wireframes, user & system workflows, or solution documentation.  
+  - **Infrastructure:** Improve the DevOps automation & security enhancements.  
+  - **Testing:** Bug reports, test coverage, or user acceptance feedback.
+
+- **Non-technical contributions** include:  
+  - **Documentation:** Write guides, create tutorials, or translate the CHT to make it accessible to more users.
+  - **Community Support** Share your knowledge by answering questions in our Support Forum, provide user feedback, or advocate for the CHT.
+  - **Collaboration: Participate in a Squad** to help tackle specific user needs or projects.
+  - **Events: Facilitation of a community event** like a Community meetup or Round-up. 
+
+This is just the beginning. To find the right opportunity for you and learn the next steps, explore the dedicated pages linked within our [Community](https://docs.communityhealthtoolkit.org/community/) page.
```

Comment:
I suggest you add that page as `relatedContent`. You can find examples on other pages.